packages/lib-classifier/src/components/Classifier/components/FieldGuide/components/FieldGuide/components/FieldGuideItemIcon/FieldGuideItemIcon.js

```diff
@@ -1,44 +1,49 @@
 import zooTheme from '@zooniverse/grommet-theme'
 import { Media } from '@zooniverse/react-components'
 import PropTypes from 'prop-types'
+import { Box } from 'grommet'
+
+export default function FieldGuideItemIcon(props) {
+  const { alt, className, height, icon, width } = props
```

Comment:
```suggestion
export default function FieldGuideItemIcon({
  alt = '',
  className = '',
  height = 100,
  icon = null,
  width = 100,
  ...props
}) {
```
Set defaults for function args and allow for optional Grommet props.